daemon/daemon.go

```diff
@@ -56,26 +56,62 @@ type Command struct {
 	PUT    ResponseFunc
 	POST   ResponseFunc
 	DELETE ResponseFunc
+	// can guest GET?
+	GuestOK bool
+	// can non-admin GET?
+	UserOK bool
```

Comment:
I went with a couple of booleans instead of an accessLevel int because it seemed simpler (no constants to have to think about). If it gets even one bit more complicated, switching to the more structured way is straightforward.